guides/create_an_app/2-channels.md

```diff
@@ -1,5 +1,161 @@
 # 2. Channels
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, suscipit a, scelerisque sed, lacinia in, mi. Cras vel lorem. Etiam pellentesque aliquet tellus. Phasellus pharetra nulla ac diam. Quisque semper justo at risus. Donec venenatis, turpis vel hendrerit interdum, dui ligula ultricies purus, sed posuere libero dui id orci. Nam congue, pede vitae dapibus aliquet, elit magna vulputate arcu, vel tempus metus leo non est. Etiam sit amet lectus quis est congue mollis. Phasellus congue lacus eget neque. Phasellus ornare, ante vitae consectetuer consequat, purus sapien ultricies dolor, et mollis pede metus eget nisi. Praesent sodales velit quis augue. Cras suscipit, urna at aliquam rhoncus, urna quam viverra nisi, in interdum massa nibh nec erat.
+Now that we have everything up and running we can start adding functionalities. The first thing we will focus on is the ability to communicate in Phoenix. This will be done via a Channel which uses websockets. Websockets create connections similar to a phone call, it means that they are always open or available. The other means of communication that are usually used on the web are more similar to sending snail mail-- a message is sent,  received and then a reply shows up. An example of this would be HTTP.
```

Comment:
The changes are highlighted in bold:

"Now that we have everything up and running we can start adding functionalities. The first thing we will focus on is the ability to communicate in Phoenix. This will be done via a Channel which uses websockets. Websockets create connections similar to a phone call, **which** means that they are always open or available. The other means of communication that are usually used on the web are more similar to sending snail mail-- a message is sent, received and then a reply shows up. An example of this would be HTTP."